olm-catalog/serverless-operator/project.yaml

```diff
@@ -4,19 +4,19 @@ project:
     # all components in `dependencies.previous` to the same versions as `dependencies` in the same PR.
     # Otherwise, the upgrade tests will not pass, as we have a different SO version with the same bundle contents.
     # Also make sure to update values under `olm.previous` by copying from `olm.replaces` and `olm.skipRange`.
-    version: 1.30.0
+    version: 1.30.1
 olm:
-    replaces: 1.29.0
-    skipRange: '>=1.29.0 <1.30.0'
+    replaces: 1.30.0
+    skipRange: '>=1.30.0 <1.30.1'
     channels:
         default: 'stable'
         list:
             - stable
             - stable-1.29
             - stable-1.30
     previous:
-        replaces: 1.28.0
-        skipRange: '>=1.28.0 <1.29.0'
+        replaces: 1.29.0
+        skipRange: '>=1.29.0 <1.30.0'
 requirements:
     kube:
         # The min version validation in `vendor/knative.dev/pkg/version/version.go`
```

Comment:
We're not bumping serving and eventing so stuff under `dependencies.previous` should have same versions as the latest:
```
dependencies:
   previous:
        serving: knative-v1.9
        eventing: knative-v1.9
        eventing_kafka_broker: knative-v1.9
```

Reply:
Downgrading Serverless should then pass.